### PR TITLE
Add new manifest for Splash new app id

### PR DESCRIPTION
**Pull request by paperManu**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: com.gitlab.sat_metalab.Splash.json
Comment (hfiguiere) [inline com.gitlab.sat_metalab.Splash.json:1]: manifest file name should match the app-id

Comment (paperManu) [reply com.gitlab.sat_metalab.Splash.json:1]: Indeed!

**Comment by hfiguiere** [comment]
bot, build xyz.splashmapper.Splash

File: xyz.splashmapper.Splash.json [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DBUILD_GENERIC_ARCH=ON"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.com/sat-mtl/tools/splash/splash.git",
+                    "tag": "0.9.36",
+                    "commit": "3923174eca6f4fff72ff24a66d50b9e69fe2433a"
```
Comment (hfiguiere) [inline xyz.splashmapper.Splash.json:128]: commit and tag don't match.

Comment (paperManu) [reply xyz.splashmapper.Splash.json]: Sorry for that, should be fixed.

**Comment by hfiguiere** [comment]
bot, build xyz.splashmapper.Splash

**Comment by paperManu** [comment]
Hum I don't know what to say, the issue seems to be a download failure for a submodule:
```
Failed to clone 'external/zmq' a second time, aborting
```
This is for both amd64 and aarch64. Any idea?

**Comment by hfiguiere** [comment]
looks like a github problem.

**Comment by paperManu** [comment]
Should we retry in a bit?

**Comment by paperManu** [comment]
Oh well, it passed in the end!

**Comment by paperManu** [comment]
I tested the generated package on my computer using X11, and it runs as expected:
```
~ ❯❯❯ flatpak run xyz.splashmapper.Splash

                     - Splash -
        - Modular multi-output video mapper -
                  - Version 0.9.36 -

2023-08-04T10:23:41 / [MESSAGE] / Link::Link - Setting up interprocess communication to shmdata
2023-08-04T10:23:41 / [MESSAGE] / No filename specified, loading default file
2023-08-04T10:23:41 / [MESSAGE] / World::addScene - Starting a Scene in another process
2023-08-04T10:23:41 / [MESSAGE] / Scene::init - GL version: 4.5
2023-08-04T10:23:41 / [MESSAGE] / Scene::init - GL vendor: AMD
2023-08-04T10:23:41 / [MESSAGE] / Scene::init - GL renderer: AMD Radeon RX Vega (vega10, LLVM 15.0.7, DRM 3.49, 6.1.0-1017-oem)
2023-08-04T10:23:41 / [MESSAGE] / Link::Link - Setting up interprocess communication to shmdata
2023-08-04T10:23:41 / [WARNING] / Factory::create - Object type colorCalibrator does not exist
2023-08-04T10:23:41 / [MESSAGE] / Window::Window - Window created successfully
2023-08-04T10:23:41 / [MESSAGE] / Window::Window - Window created successfully
ALSA lib ../../../src/pcm/pcm_direct.c:2045:(snd1_pcm_direct_parse_open_conf) The field ipc_gid must be a valid group (create group audio)
ALSA lib ../../../src/pcm/pcm_direct.c:2045:(snd1_pcm_direct_parse_open_conf) The field ipc_gid must be a valid group (create group audio)
ALSA lib ../../../src/pcm/pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.rear
ALSA lib ../../../src/pcm/pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.center_lfe
ALSA lib ../../../src/pcm/pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM cards.pcm.side
ALSA lib ../../oss/pcm_oss.c:397:(_snd_pcm_oss_open) Cannot open device /dev/dsp
ALSA lib ../../oss/pcm_oss.c:397:(_snd_pcm_oss_open) Cannot open device /dev/dsp
ALSA lib ../../src/confmisc.c:160:(snd_config_get_card) Invalid field card
ALSA lib ../../usb_stream/pcm_usb_stream.c:482:(_snd_pcm_usb_stream_open) Invalid card 'card'
ALSA lib ../../src/confmisc.c:160:(snd_config_get_card) Invalid field card
ALSA lib ../../usb_stream/pcm_usb_stream.c:482:(_snd_pcm_usb_stream_open) Invalid card 'card'
ALSA lib ../../../src/pcm/pcm_direct.c:2045:(snd1_pcm_direct_parse_open_conf) The field ipc_gid must be a valid group (create group audio)
2023-08-04T10:23:42 / [MESSAGE] / Sound_Engine::getDevice - Connected to device: default
2023-08-04T10:23:42 / [MESSAGE] / Sound_Engine::startStream - Successfully opened PortAudio stream
2023-08-04T10:23:42 / [MESSAGE] / LtcClock::LtcClock - Input clock enabled
2023-08-04T10:23:46 / [MESSAGE] / LtcClock::~LtcClock - Input clock disabled
```

**Pull request closed without merging**